features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/utils/ColorConstants.kt

```diff
@@ -8,10 +8,16 @@ package com.datadog.android.sessionreplay.utils
 
 internal const val ALPHA_SHIFT_ANDROID = 24
 internal const val ALPHA_SHIFT_WEB = 8
-internal const val MAX_ALPHA_VALUE = 0xFF
 
+internal const val MAX_ALPHA_VALUE = 0xFF
 internal const val WEB_COLOR_STR_LENGTH = 8
 
 internal const val MASK_ALPHA = 0xFF000000L
 internal const val MASK_RGB = 0x00FFFFFFL
 internal const val MASK_COLOR = 0xFFFFFFFFL
+
+/** The value corresponding to an opaque Alpha. */
+const val OPAQUE_ALPHA_VALUE: Int = MAX_ALPHA_VALUE
+
+/** The value corresponding to a 25% opaque Alpha. */
+const val PARTIALLY_OPAQUE_ALPHA_VALUE: Int = 0x44
```

Comment:
a random mathematical comment here, 0x3f or 0x40 seems closer to 25% of opaque alpha.(Just wondering how 0x44 is calculated, although it hardly has any impact)

Reply:
Good point thanks